docs/quick-start.md

```diff
@@ -23,7 +23,7 @@ $ npx @featurevisor/cli <command>
 
 ## Initialize your project
 
-This is maent be a completely separate repository from your application code. The idea is to be able to decouple your application deployments from releasing your features.
+This is meant be a completely separate repository from your application code. The idea is to be able to decouple your application deployments from releasing your features.
```

Comment:
thanks for this fix, @caioquirino!

I think we can make it `This is meant to be ` here.